lib/heart_of_a_developer.rb

```diff
@@ -37,8 +37,9 @@ class HeartOfADeveloper
   ].freeze
   LINE_LENGTHS = [25, 23, 19, 15].freeze
 
-  def self.speak
-    puts illustration(partition_phrase(sample_phrase))
+  def self.speak(user_phrase = nil)
```

Comment:
What if we use default arguments a bit more here? Like:
```rb
   def self.speak(phrase_to_output = sample_phrase)
     puts illustration(partition_phrase(phrase_to_output))
   end
```

Reply:
Sure, thats certainly cleaner. Updated the pr based on feedback.